posts/how-to-build-a -travel-route-planning-application.md

```diff
@@ -0,0 +1,231 @@
+---
+permalink: /blog/:path/
+layout: blog-subpages
+
+title: How to Build a Travel Route Planning Application with Memgraph and Cypher
+description: In this tutorial, you will build a simple graph application with Memgraph and Cypher to navigate the European road network and learn how to use some of the most popular graph algorithms used to solve routing problems including
+breadth-first search (BFS), and Dijkstra's algorithm.
+status: published
+header-image: "https://i.imgur.com/WN8Dwsb.jpg "
+category: [Tutorial]
+
+sitemap: true
+---
+
+# How to Build a Travel Route Planning Application with Memgraph and Cypher
+
+## Introduction
+One of the most common applications of graph traversal algorithms is route planning and optimization problems. These can vary from simple things such as finding the shortest path from a starting point to a destination to more complex scenarios where it is required to pass a particular intermediate node (e.g. A specific city) and satisfy multiple conditions(e.g. Limited number of stops) before reaching the final destination. Some practical examples include:
+* Logistics and supply chain optimization – Planning effective routes to minimize cost, increase vehicle utilization and improve operations by taking into account things like travel distance, vehicle capacity, road tolls, fuel consumption, and border taxes.
+* Travel route planning – Finding the quickest way to get from point A to point B taking into account things like traffic, road closures, and construction.
+* Network optimization – Improving communications and information transfer in telephone and cellular networks by finding optimal paths based on factors such as bandwidth.
+
+In this tutorial, we will build a simple graph application with Memgraph and Cypher to navigate the European road network and learn how to use some of the most popular graph algorithms used to solve routing problems including
+[breadth-first search (BFS)](https://en.wikipedia.org/wiki/Breadth-first_search), and [Dijkstra's algorithm](https://en.wikipedia.org/wiki/Dijkstra%27s_algorithm).
+
+## Prerequisites
+
+To follow along, you will need the following:
+* A local installation of Memgraph. You can refer to [the Memgraph documentation](https://docs.memgraph.com/).
```

Comment:
Maybe this would be a better link: `https://docs.memgraph.com/memgraph/quick-start#installation`